recipes/openai/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "openai" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f514098dece360dcee8fe1b33eed876955af7f2e3028362a54f00f8522b71408
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py<27]
```

Comment:
I'm getting this from https://github.com/openai/openai-python/blob/f00f489455ccbf4feb4bcf65f89d497fc4dd31ff/setup.py#L20, is that correct?

Reply:
Odds are it is simpler to just make this a python noarch with python >=3.6 and ignore py27. support

Reply:
noarch is not possible because of `scripts=["bin/openai"],` at https://github.com/openai/openai-python/blob/f00f489455ccbf4feb4bcf65f89d497fc4dd31ff/setup.py#L21

From how I understand the docs, having a `scripts` element disqualifies a recipe from being noarch?